jenkins_jobs/update_versions_readme/script.sh

```diff
@@ -0,0 +1,9 @@
+python host-os.py \
+  --verbose \
+  update-versions-readme \
```

Comment:
Why create a update-versions-readme subcommand instead of doing this in existing update-versions subcommand?